<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RevanchasLTDialog</class>
 <widget class="QDialog" name="RevanchasLTDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Revanchas LT - An√°lisis Topogr√°fico</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="wall_label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Muro seleccionado: [None]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="alignment_group">
     <property name="title">
      <string>Informaci√≥n de Alineaci√≥n</string>
     </property>
     <layout class="QVBoxLayout" name="alignment_layout">
      <item>
       <widget class="QLabel" name="alignment_info_label">
        <property name="text">
         <string>Seleccione un muro para ver informaci√≥n de alineaci√≥n</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dem_group">
     <property name="title">
      <string>Archivo DEM</string>
     </property>
     <layout class="QVBoxLayout" name="dem_layout">
      <item>
       <layout class="QHBoxLayout" name="dem_file_layout">
        <item>
         <widget class="QLabel" name="dem_path_label">
          <property name="text">
           <string>DEM: [No seleccionado]</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="browse_dem_button">
          <property name="text">
           <string>Examinar...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="dem_info_label">
        <property name="text">
         <string>Seleccione un archivo DEM (.asc) para ver informaci√≥n</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ecw_group">
     <property name="title">
      <string>Archivo Ortomosaico (ECW)</string>
     </property>
     <layout class="QVBoxLayout" name="ecw_layout">
      <item>
       <layout class="QHBoxLayout" name="ecw_file_layout">
        <item>
         <widget class="QLabel" name="ecw_path_label">
          <property name="text">
           <string>ECW: [No seleccionado]</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="browse_ecw_button">
          <property name="text">
           <string>Examinar...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="ecw_info_label">
        <property name="text">
         <string>Seleccione un archivo ortomosaico (.ecw) para ver informaci√≥n</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="profiles_group">
     <property name="title">
      <string>üéØ An√°lisis Interactivo de Perfiles</string>
     </property>
     <layout class="QVBoxLayout" name="profiles_layout">
      <item>
       <widget class="QPushButton" name="generate_profiles_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>üöÄ Generar y Visualizar Perfiles</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Genera perfiles topogr√°ficos cada 20m y abre el visualizador interactivo con herramientas de medici√≥n</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="profiles_info_label">
        <property name="text">
         <string>‚Ä¢ Perfiles cada 20m con 80m de ancho de an√°lisis
‚Ä¢ Navegaci√≥n interactiva entre estaciones (PK)
‚Ä¢ Herramientas de medici√≥n: Cota Coronamiento y Ancho
‚Ä¢ Visualizaci√≥n profesional con grilla de coordenadas</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RevanchasLTDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RevanchasLTDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>